config/tests/fixtures.rs

```diff
@@ -172,7 +172,7 @@ fn minimal_config_snapshot() {
                 idle_time: 30s,
             },
             logger: Logger {
-                level: INFO,
+                level: info,
                 format: Full,
```

Comment:
I notice a difference in casing

Reply:
let's open an issue to standardize this

Reply:
you can still specify TRACE, DEBUG, INFO, WARN, ERROR

lower-case affect only serialisation

Reply:
I meant that we have `info` for `level` on the one hand and `Full` for `format` on the other hand. I don't mind having all lowercase or all uppercase, but I don't like seeing cammel case. I wasn't complaining about your change

Reply:
i see, i thought you said you noticed the difference between `info` and `INFO` :)

Reply:
Created #4840